### PR TITLE
note about 3rd party resources

### LINKED ISSUE #183: Bootstrap for Sass javascript not included. Jquery included twice
I did a default 'yo webapp' and selected Bootstrap for Sass.
I noticed that none of the bootstrap javascript is working. For instance If I make a default collapsible navbar, it does not open on mobile unless i explicitly include collapse.js in the html.

http://getbootstrap.com/components/#navbar-default

Also jquery is included once in the body tag of index.html and again in main.js.

### PR DESCRIPTION
**Pull request by eddiemonge**
Fixes #183

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -20,8 +20,20 @@ For more information on what `generator-webapp` can do for you, take a look at t
...
+#### 3rd Party Resources
+
+3rd party resources are managed with [bower-install](https://github.com/stephenplusplus/grunt-bower-install). Add new resources using Bower and then run the Grunt task to load them:
+
+```bash
+  bower install jquery --save
+  grunt bower-install
+```
+
+This works if the package author has followed the Bower spec. If the files are not automatically added to your index.html, check with the package's repo for support and/or file an issue with them to have it updated.
```
Comment (sindresorhus) [inline readme.md]: Should also be explicit about them having to manually add it and where.

**Comment by eddiemonge** [comment]
When I think of Bower I think of resources. When I hear dependencies I think of npm but thats just me. I'm ok with changing it if you insist and/or anyone else agrees

**Comment by sindresorhus** [comment]
They're really both, but resources can be so much more. I think dependencies are more explicit.

**Comment by eddiemonge** [comment]
Updated

**Comment by sindresorhus** [comment]
Thanks :) Happy new years

**Pull request merged**